src/z3c/pt/tests/test_doctests.py

```diff
@@ -18,31 +18,40 @@
 import zope.configuration.xmlconfig
 import z3c.pt
 
-OPTIONFLAGS = (doctest.ELLIPSIS
-               | doctest.NORMALIZE_WHITESPACE)
+OPTIONFLAGS = doctest.ELLIPSIS | doctest.NORMALIZE_WHITESPACE
 
 
 def setUp(suite):
     zope.component.testing.setUp(suite)
-    zope.configuration.xmlconfig.XMLConfig('configure.zcml', z3c.pt)()
+    zope.configuration.xmlconfig.XMLConfig("configure.zcml", z3c.pt)()
 
 
 def test_suite():
-    filesuites = ('README.rst',)
-    testsuites = 'z3c.pt.expressions', 'z3c.pt.namespaces'
+    filesuites = ("README.rst",)
+    testsuites = "z3c.pt.expressions", "z3c.pt.namespaces"
 
     return unittest.TestSuite(
-        [doctest.DocFileSuite(
-            filesuite, optionflags=OPTIONFLAGS,
-            setUp=setUp, tearDown=zope.component.testing.tearDown,
-            package="z3c.pt")
-         for filesuite in filesuites]
-        +
-        [doctest.DocTestSuite(
-            testsuite, optionflags=OPTIONFLAGS,
-            setUp=setUp, tearDown=zope.component.testing.tearDown)
-         for testsuite in testsuites]
-        )
-
-if __name__ == '__main__':
-    unittest.main(defaultTest='test_suite')
+        [
+            doctest.DocFileSuite(
+                filesuite,
+                optionflags=OPTIONFLAGS,
+                setUp=setUp,
+                tearDown=zope.component.testing.tearDown,
+                package="z3c.pt",
+            )
+            for filesuite in filesuites
+        ]
+        + [
+            doctest.DocTestSuite(
+                testsuite,
+                optionflags=OPTIONFLAGS,
+                setUp=setUp,
+                tearDown=zope.component.testing.tearDown,
+            )
+            for testsuite in testsuites
+        ]
```

Comment:
IIRC both DocFileSuite and DocTestSuite accept multiple positional args, so `*filesuites`/`*testsuites` would work, if we put them after the kwargs.

Reply:
If I understand the [documentation](https://docs.python.org/2.7/library/doctest.html#doctest.DocTestSuite) correctly `DocTestSuite` only accepts a single module name. I refactored the code to get rid of the `for` loops.

Reply:
D'oh, when will I learn not to trust my memory?

Reply:
At least I hedged my bets with that "IIRC" :)